ci-operator/step-registry/sandboxed-containers-operator/create-prowjob/sandboxed-containers-operator-create-prowjob-commands.sh

```diff
@@ -407,8 +407,8 @@ if [[ ${file_size} -lt 1000 ]]; then
 fi
 
 # Basic YAML syntax validation if yq is available
-if command -v yq &> /dev/null; then
-    echo "Validating YAML: yq eval '.' "${OUTPUT_FILE}"
+if command -v yq 2&> /dev/null; then
```

Comment:
This fixes an extra quote, but it introduces a bug as `2&>` redirection is invalid. Do you also want to redirect only stderr (then the commit message should also reflect this change).